tests/pages/services/TasksTable-cy.js

```diff
@@ -232,6 +232,14 @@ describe("Tasks Table", function() {
       cy.get("table tr td.task-table-column-name").should("to.have.length", 4);
     });
 
+    it("Filters tasks that are in failed state", function() {
+      cy.get(".form-control.filter-input-text")
+        .type("{selectall}{backspace}")
+        .type("{selectall}{backspace}");
+      cy.get(".form-control.filter-input-text").type("is:failed");
```

Comment:
Test is checking for `is:failed` but in the code it is `is:failure`.
Let's please change it to `is:failed` in the code as well

Reply:
Good catch, I will update it.

Reply:
I think the `is:failure` is correct, so we should update the test.

Reply:
We use verbiage: 
`is:active`
`is:killed`
`is:completed`
should be
`is:failed` following the logic

Reply:
`failure` already existed on master - https://github.com/dcos/dcos-ui/blob/master/plugins/services/src/js/constants/TaskStates.js#L45 , this is why I used it.